crates/jupyter-protocol/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "jupyter-protocol"
-version = "0.8.1"
+version = "0.9.0"
```

Comment:
In the future, I'll take a suggested version bump as I use `cargo-release` to ship this. Check out `RELEASING.md` in the repo for details.